senders/telegram/send_test.go

```diff
@@ -168,3 +170,50 @@ func TestPrepareAlbum(t *testing.T) {
 		})
 	})
 }
+
+func TestCheckBrokenContactError(t *testing.T) {
+	logger, _ := logging.ConfigureLog("stdout", "debug", "test", true)
+	Convey("Check broken contact error", t, func() {
```

Comment:
```suggestion
	Convey("During checking broken contact error", t, func() {
```

Reply:
I like original wording more, I feel it's more correct. How big is the value in difference with your proposed phrase? :)

Reply:
I wanted to follow the test framework author's [examples](https://github.com/smartystreets/goconvey/tree/master/examples), but if you want to leave it like that, then ok :)